Package.swift

```diff
@@ -0,0 +1,57 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
```

Comment:
This is still failing the swift-format check.   In particular it wants:

```swift
 import PackageDescription
+
 let package = Package(
```

Reply:
fixed in the latest commit. Thanks.